[2025-06-29 22:33:59,169.169] _internal.py -> _log line:97 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-06-29 22:33:59,170.170] _internal.py -> _log line:97 [INFO]: [33mPress CTRL+C to quit[0m
[2025-06-29 22:33:59,171.171] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 22:34:00,460.460] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 22:34:00,473.473] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 22:34:11,289.289] base.py -> _exec_single_context line:1896 [INFO]: select pg_catalog.version()
[2025-06-29 22:34:11,290.290] base.py -> _exec_single_context line:1901 [INFO]: [raw sql] {}
[2025-06-29 22:34:11,293.293] base.py -> _exec_single_context line:1896 [INFO]: select current_schema()
[2025-06-29 22:34:11,293.293] base.py -> _exec_single_context line:1901 [INFO]: [raw sql] {}
[2025-06-29 22:34:11,294.294] base.py -> _exec_single_context line:1896 [INFO]: show standard_conforming_strings
[2025-06-29 22:34:11,295.295] base.py -> _exec_single_context line:1901 [INFO]: [raw sql] {}
[2025-06-29 22:34:11,295.295] base.py -> _begin_impl line:1097 [INFO]: BEGIN (implicit)
[2025-06-29 22:34:11,297.297] base.py -> _exec_single_context line:1896 [INFO]: SELECT dataset.id AS dataset_id, dataset.account_id AS dataset_account_id, dataset.name AS dataset_name, dataset.icon AS dataset_icon, dataset.description AS dataset_description, dataset.updated_at AS dataset_updated_at, dataset.created_at AS dataset_created_at 
FROM dataset 
WHERE dataset.id = %(pk_1)s::UUID
[2025-06-29 22:34:11,297.297] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00021s] {'pk_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:34:11,303.303] base.py -> _exec_single_context line:1896 [INFO]: SELECT count(document.id) AS count_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:34:11,303.303] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00016s] {'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:34:11,307.307] base.py -> _exec_single_context line:1896 [INFO]: SELECT coalesce(sum(segment.hit_count), %(coalesce_2)s) AS coalesce_1 
FROM segment 
WHERE segment.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:34:11,307.307] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00014s] {'coalesce_2': 0, 'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:34:11,313.313] base.py -> _exec_single_context line:1896 [INFO]: SELECT count(app_dataset_join.id) AS count_1 
FROM app_dataset_join 
WHERE app_dataset_join.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:34:11,313.313] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00018s] {'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:34:11,315.315] base.py -> _exec_single_context line:1896 [INFO]: SELECT coalesce(sum(document.character_count), %(coalesce_2)s) AS coalesce_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:34:11,315.315] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00014s] {'coalesce_2': 0, 'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:34:11,316.316] base.py -> _rollback_impl line:1123 [INFO]: ROLLBACK
[2025-06-29 22:34:11,316.316] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 22:34:11] "GET /datasets/6c5e3fd6-6307-4eb5-b017-895b25ce4a1c HTTP/1.1" 200 -
[2025-06-29 22:35:11,440.440] base.py -> _begin_impl line:1097 [INFO]: BEGIN (implicit)
[2025-06-29 22:35:11,441.441] base.py -> _exec_single_context line:1896 [INFO]: SELECT dataset.id AS dataset_id, dataset.account_id AS dataset_account_id, dataset.name AS dataset_name, dataset.icon AS dataset_icon, dataset.description AS dataset_description, dataset.updated_at AS dataset_updated_at, dataset.created_at AS dataset_created_at 
FROM dataset 
WHERE dataset.id = %(pk_1)s::UUID
[2025-06-29 22:35:11,441.441] base.py -> _exec_single_context line:1901 [INFO]: [cached since 60.14s ago] {'pk_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:35:11,444.444] base.py -> _exec_single_context line:1896 [INFO]: SELECT count(document.id) AS count_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:35:11,444.444] base.py -> _exec_single_context line:1901 [INFO]: [cached since 60.14s ago] {'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:35:11,444.444] base.py -> _exec_single_context line:1896 [INFO]: SELECT coalesce(sum(segment.hit_count), %(coalesce_2)s) AS coalesce_1 
FROM segment 
WHERE segment.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:35:11,444.444] base.py -> _exec_single_context line:1901 [INFO]: [cached since 60.14s ago] {'coalesce_2': 0, 'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:35:11,445.445] base.py -> _exec_single_context line:1896 [INFO]: SELECT count(app_dataset_join.id) AS count_1 
FROM app_dataset_join 
WHERE app_dataset_join.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:35:11,445.445] base.py -> _exec_single_context line:1901 [INFO]: [cached since 60.13s ago] {'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:35:11,445.445] base.py -> _exec_single_context line:1896 [INFO]: SELECT coalesce(sum(document.character_count), %(coalesce_2)s) AS coalesce_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:35:11,445.445] base.py -> _exec_single_context line:1901 [INFO]: [cached since 60.13s ago] {'coalesce_2': 0, 'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:35:11,446.446] base.py -> _rollback_impl line:1123 [INFO]: ROLLBACK
[2025-06-29 22:35:11,447.447] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 22:35:11] "GET /datasets/6c5e3fd6-6307-4eb5-b017-895b25ce4a1c HTTP/1.1" 200 -
[2025-06-29 22:39:50,167.167] _internal.py -> _log line:97 [INFO]:  * Detected change in '/Users/icourt1/Desktop/code/llmops/llmops-api/internal/handler/dataset_handler.py', reloading
[2025-06-29 22:39:50,371.371] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 22:39:51,945.945] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 22:39:51,956.956] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 22:39:56,224.224] base.py -> _exec_single_context line:1896 [INFO]: select pg_catalog.version()
[2025-06-29 22:39:56,224.224] base.py -> _exec_single_context line:1901 [INFO]: [raw sql] {}
[2025-06-29 22:39:56,225.225] base.py -> _exec_single_context line:1896 [INFO]: select current_schema()
[2025-06-29 22:39:56,225.225] base.py -> _exec_single_context line:1901 [INFO]: [raw sql] {}
[2025-06-29 22:39:56,226.226] base.py -> _exec_single_context line:1896 [INFO]: show standard_conforming_strings
[2025-06-29 22:39:56,226.226] base.py -> _exec_single_context line:1901 [INFO]: [raw sql] {}
[2025-06-29 22:39:56,226.226] base.py -> _begin_impl line:1097 [INFO]: BEGIN (implicit)
[2025-06-29 22:39:56,228.228] base.py -> _exec_single_context line:1896 [INFO]: SELECT dataset.id AS dataset_id, dataset.account_id AS dataset_account_id, dataset.name AS dataset_name, dataset.icon AS dataset_icon, dataset.description AS dataset_description, dataset.updated_at AS dataset_updated_at, dataset.created_at AS dataset_created_at 
FROM dataset 
WHERE dataset.id = %(pk_1)s::UUID
[2025-06-29 22:39:56,228.228] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00012s] {'pk_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:39:56,230.230] dataset_handler.py -> get_dataset line:58 [ERROR]: æµ‹è¯•log
[2025-06-29 22:39:56,232.232] base.py -> _exec_single_context line:1896 [INFO]: SELECT count(document.id) AS count_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:39:56,233.233] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00017s] {'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:39:56,234.234] base.py -> _exec_single_context line:1896 [INFO]: SELECT coalesce(sum(segment.hit_count), %(coalesce_2)s) AS coalesce_1 
FROM segment 
WHERE segment.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:39:56,234.234] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00012s] {'coalesce_2': 0, 'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:39:56,237.237] base.py -> _exec_single_context line:1896 [INFO]: SELECT count(app_dataset_join.id) AS count_1 
FROM app_dataset_join 
WHERE app_dataset_join.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:39:56,237.237] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00012s] {'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:39:56,238.238] base.py -> _exec_single_context line:1896 [INFO]: SELECT coalesce(sum(document.character_count), %(coalesce_2)s) AS coalesce_1 
FROM document 
WHERE document.dataset_id = %(dataset_id_1)s::UUID
[2025-06-29 22:39:56,239.239] base.py -> _exec_single_context line:1901 [INFO]: [generated in 0.00012s] {'coalesce_2': 0, 'dataset_id_1': UUID('6c5e3fd6-6307-4eb5-b017-895b25ce4a1c')}
[2025-06-29 22:39:56,239.239] base.py -> _rollback_impl line:1123 [INFO]: ROLLBACK
[2025-06-29 22:39:56,240.240] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 22:39:56] "GET /datasets/6c5e3fd6-6307-4eb5-b017-895b25ce4a1c HTTP/1.1" 200 -
[2025-06-29 22:40:23,008.008] _internal.py -> _log line:97 [INFO]:  * Detected change in '/Users/icourt1/Desktop/code/llmops/llmops-api/internal/handler/dataset_handler.py', reloading
[2025-06-29 22:40:23,241.241] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 22:40:24,622.622] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 22:40:24,632.632] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:20:32,501.501] _internal.py -> _log line:97 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-06-29 23:20:32,501.501] _internal.py -> _log line:97 [INFO]: [33mPress CTRL+C to quit[0m
[2025-06-29 23:20:32,502.502] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:20:36,320.320] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:20:36,343.343] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:23:23,267.267] _internal.py -> _log line:97 [INFO]:  * Detected change in '/Users/icourt1/Desktop/code/llmops/llmops-api/app/http/moudle.py', reloading
[2025-06-29 23:23:23,547.547] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:23:25,092.092] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:23:25,103.103] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:24:15,878.878] _internal.py -> _log line:97 [INFO]:  * Detected change in '/Users/icourt1/Desktop/code/llmops/llmops-api/config/config.py', reloading
[2025-06-29 23:24:16,186.186] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:24:17,839.839] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:24:17,855.855] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:33:09,507.507] _internal.py -> _log line:97 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-06-29 23:33:09,508.508] _internal.py -> _log line:97 [INFO]: [33mPress CTRL+C to quit[0m
[2025-06-29 23:33:09,509.509] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:33:11,286.286] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:33:11,302.302] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:33:23,614.614] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 23:33:23] "[35m[1mGET /ping HTTP/1.1[0m" 500 -
[2025-06-29 23:45:52,990.990] _internal.py -> _log line:97 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-06-29 23:45:52,991.991] _internal.py -> _log line:97 [INFO]: [33mPress CTRL+C to quit[0m
[2025-06-29 23:45:52,992.992] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:45:54,524.524] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:45:54,542.542] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:46:05,692.692] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 23:46:05] "[35m[1mGET /ping HTTP/1.1[0m" 500 -
[2025-06-29 23:47:42,408.408] _internal.py -> _log line:97 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-06-29 23:47:42,409.409] _internal.py -> _log line:97 [INFO]: [33mPress CTRL+C to quit[0m
[2025-06-29 23:47:42,414.414] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:47:54,883.883] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:47:55,101.101] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:47:56,281.281] http.py -> _register_error_handler line:70 [ERROR]: An error occurred: Object of type bytes is not JSON serializable
Traceback (most recent call last):
  File "/Users/icourt1/.pyenv/versions/llmops-env/lib/python3.10/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/icourt1/.pyenv/versions/llmops-env/lib/python3.10/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/icourt1/Desktop/code/llmops/llmops-api/internal/handler/app_handler.py", line 120, in ping
    return success_json({"ping": self.redis_client.get("name")})
  File "/Users/icourt1/Desktop/code/llmops/llmops-api/pkg/response/response.py", line 31, in success_json
    return json(Response(code=HttpCode.SUCCESS, message="", data=data))
  File "/Users/icourt1/Desktop/code/llmops/llmops-api/pkg/response/response.py", line 26, in json
    return jsonify(data), 200
  File "/Users/icourt1/.pyenv/versions/llmops-env/lib/python3.10/site-packages/flask/json/__init__.py", line 170, in jsonify
    return current_app.json.response(*args, **kwargs)  # type: ignore[return-value]
  File "/Users/icourt1/.pyenv/versions/llmops-env/lib/python3.10/site-packages/flask/json/provider.py", line 214, in response
    f"{self.dumps(obj, **dump_args)}\n", mimetype=self.mimetype
  File "/Users/icourt1/.pyenv/versions/llmops-env/lib/python3.10/site-packages/flask/json/provider.py", line 179, in dumps
    return json.dumps(obj, **kwargs)
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/encoder.py", line 439, in _iterencode
    yield from _iterencode(o, _current_indent_level)
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/encoder.py", line 431, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/encoder.py", line 405, in _iterencode_dict
    yield from chunks
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/encoder.py", line 405, in _iterencode_dict
    yield from chunks
  File "/Users/icourt1/.pyenv/versions/3.10.16/lib/python3.10/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/icourt1/.pyenv/versions/llmops-env/lib/python3.10/site-packages/flask/json/provider.py", line 121, in _default
    raise TypeError(f"Object of type {type(o).__name__} is not JSON serializable")
TypeError: Object of type bytes is not JSON serializable
[2025-06-29 23:47:56,636.636] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 23:47:56] "[35m[1mGET /ping HTTP/1.1[0m" 500 -
[2025-06-29 23:49:55,126.126] _internal.py -> _log line:97 [INFO]: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
[2025-06-29 23:49:55,127.127] _internal.py -> _log line:97 [INFO]: [33mPress CTRL+C to quit[0m
[2025-06-29 23:49:55,130.130] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:50:08,714.714] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:50:08,799.799] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:50:10,614.614] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 23:50:10] "GET /ping HTTP/1.1" 200 -
[2025-06-29 23:50:35,656.656] _internal.py -> _log line:97 [INFO]:  * Detected change in '/Users/icourt1/Desktop/code/llmops/llmops-api/internal/handler/app_handler.py', reloading
[2025-06-29 23:50:42,495.495] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:50:52,699.699] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:50:52,763.763] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
[2025-06-29 23:50:55,787.787] _internal.py -> _log line:97 [INFO]: 127.0.0.1 - - [29/Jun/2025 23:50:55] "GET /ping HTTP/1.1" 200 -
[2025-06-29 23:51:08,649.649] _internal.py -> _log line:97 [INFO]:  * Detected change in '/Users/icourt1/Desktop/code/llmops/llmops-api/internal/handler/app_handler.py', reloading
[2025-06-29 23:51:11,974.974] _internal.py -> _log line:97 [INFO]:  * Restarting with stat
[2025-06-29 23:51:20,357.357] _internal.py -> _log line:97 [WARNING]:  * Debugger is active!
[2025-06-29 23:51:20,426.426] _internal.py -> _log line:97 [INFO]:  * Debugger PIN: 133-334-152
